cvs diff: Diffing svx
cvs diff: Diffing svx/doc
cvs diff: Diffing svx/doc/UML
cvs diff: Diffing svx/inc
cvs diff: Diffing svx/inc/pch
cvs diff: Diffing svx/inc/svx
cvs diff: Diffing svx/inc/svx/sdr
cvs diff: Diffing svx/inc/svx/sdr/animation
cvs diff: Diffing svx/inc/svx/sdr/contact
cvs diff: Diffing svx/inc/svx/sdr/event
cvs diff: Diffing svx/inc/svx/sdr/mixer
cvs diff: Diffing svx/inc/svx/sdr/overlay
cvs diff: Diffing svx/inc/svx/sdr/properties
cvs diff: Diffing svx/prj
cvs diff: Diffing svx/qa
cvs diff: Diffing svx/qa/unoapi
cvs diff: Diffing svx/qa/unoapi/testdocuments
cvs diff: Diffing svx/sdi
cvs diff: Diffing svx/source
cvs diff: Diffing svx/source/accessibility
cvs diff: Diffing svx/source/customshapes
cvs diff: Diffing svx/source/dialog
Index: svx/source/dialog/makefile.mk
===================================================================
RCS file: /cvs/graphics/svx/source/dialog/makefile.mk,v
retrieving revision 1.59
retrieving revision 1.59.122.1
diff -p -u -u -p -b -w -B -r1.59 -r1.59.122.1
--- svx/source/dialog/makefile.mk	27 Nov 2007 15:28:28 -0000	1.59
+++ svx/source/dialog/makefile.mk	12 Feb 2008 12:43:51 -0000	1.59.122.1
@@ -44,6 +44,10 @@ ENABLE_EXCEPTIONS=TRUE
 .INCLUDE :  settings.mk
 .INCLUDE :  $(PRJ)$/util$/makefile.pmk
 
+.IF "$(ENABLE_LAYOUT)" == "TRUE"
+CFLAGS+= -DENABLE_LAYOUT=1 -I../$(PRJ)/layout/inc -I../$(PRJ)/layout/$(INPATH)/inc
+.ENDIF # ENABLE_LAYOUT == TRUE
+
 # --- Files --------------------------------------------------------
 
 SRS1NAME=dialogs
Index: svx/source/dialog/zoom.cxx
===================================================================
RCS file: /cvs/graphics/svx/source/dialog/zoom.cxx,v
retrieving revision 1.9
retrieving revision 1.9.302.1
diff -p -u -u -p -b -w -B -r1.9 -r1.9.302.1
--- svx/source/dialog/zoom.cxx	27 Jun 2007 17:52:39 -0000	1.9
+++ svx/source/dialog/zoom.cxx	12 Feb 2008 12:43:51 -0000	1.9.302.1
@@ -61,7 +61,9 @@
 #define _SVX_ZOOM_CXX
 
 #include <svx/dialogs.hrc>
+#if !ENABLE_LAYOUT
 #include "zoom.hrc"
+#endif /* !ENABLE_LAYOUT */
 
 #include "zoom.hxx"
 #include <svx/zoomitem.hxx>
@@ -70,6 +72,9 @@
 #ifndef _SVX_ZOOM_DEF_HXX
 #include "zoom_def.hxx"
 #endif
+
+#include <layout/layout-pre.hxx>
+
 // static ----------------------------------------------------------------
 
 static USHORT pRanges[] =
Index: svx/source/dialog/zoom.hxx
===================================================================
RCS file: /cvs/graphics/svx/source/dialog/zoom.hxx,v
retrieving revision 1.3
retrieving revision 1.3.1188.1
diff -p -u -u -p -b -w -B -r1.3 -r1.3.1188.1
--- svx/source/dialog/zoom.hxx	8 Sep 2005 22:23:41 -0000	1.3
+++ svx/source/dialog/zoom.hxx	12 Feb 2008 12:43:51 -0000	1.3.1188.1
@@ -37,6 +37,7 @@
 
 // include ---------------------------------------------------------------
 
+#include <layout/layout.hxx>
 #ifndef _BASEDLGS_HXX //autogen wg. SfxModalDialog
 #include <sfx2/basedlgs.hxx>
 #endif
@@ -70,6 +71,8 @@
 	SvxZoomItem	<SID_ATTR_ZOOM>
 */
 
+#include <layout/layout-pre.hxx>
+
 class SvxZoomDialog : public SfxModalDialog
 {
 private:
@@ -114,6 +117,7 @@ public:
 	void				SetSpinSize( USHORT nNewSpin );
 };
 
+#include <layout/layout-post.hxx>
 
 #endif
 
cvs diff: Diffing svx/source/editeng
cvs diff: Diffing svx/source/engine3d
cvs diff: Diffing svx/source/fmcomp
cvs diff: Diffing svx/source/form
cvs diff: Diffing svx/source/gallery2
cvs diff: Diffing svx/source/gengal
cvs diff: Diffing svx/source/inc
cvs diff: Diffing svx/source/init
cvs diff: Diffing svx/source/intro
cvs diff: Diffing svx/source/items
cvs diff: Diffing svx/source/mnuctrls
cvs diff: Diffing svx/source/msfilter
cvs diff: Diffing svx/source/options
cvs diff: Diffing svx/source/outliner
cvs diff: Diffing svx/source/sdr
cvs diff: Diffing svx/source/sdr/animation
cvs diff: Diffing svx/source/sdr/contact
cvs diff: Diffing svx/source/sdr/event
cvs diff: Diffing svx/source/sdr/mixer
cvs diff: Diffing svx/source/sdr/overlay
cvs diff: Diffing svx/source/sdr/properties
cvs diff: Diffing svx/source/smarttags
cvs diff: Diffing svx/source/src
cvs diff: Diffing svx/source/stbctrls
cvs diff: Diffing svx/source/svdraw
cvs diff: Diffing svx/source/svrtf
cvs diff: Diffing svx/source/svxlink
cvs diff: Diffing svx/source/tbxctrls
cvs diff: Diffing svx/source/toolbars
cvs diff: Diffing svx/source/unodialogs
cvs diff: Diffing svx/source/unodialogs/inc
cvs diff: Diffing svx/source/unodialogs/textconversiondlgs
cvs diff: Diffing svx/source/unodraw
cvs diff: Diffing svx/source/unoedit
cvs diff: Diffing svx/source/unogallery
cvs diff: Diffing svx/source/xml
cvs diff: Diffing svx/source/xoutdev
cvs diff: Diffing svx/uiconfig
cvs diff: Diffing svx/uiconfig/accelerator
cvs diff: Diffing svx/uiconfig/accelerator/en-US
cvs diff: Diffing svx/uiconfig/accelerator/es
cvs diff: Diffing svx/util
Index: svx/util/makefile.mk
===================================================================
RCS file: /cvs/graphics/svx/util/makefile.mk,v
retrieving revision 1.53
retrieving revision 1.53.136.1
diff -p -u -u -p -b -w -B -r1.53 -r1.53.136.1
--- svx/util/makefile.mk	2 Nov 2007 12:20:40 -0000	1.53
+++ svx/util/makefile.mk	12 Feb 2008 12:43:52 -0000	1.53.136.1
@@ -187,6 +187,11 @@ SHL2STDLIBS+= \
              $(ADVAPI32LIB)
 .ENDIF # WNT
 
+.IF "$(ENABLE_LAYOUT)" == "TRUE"
+LINKFLAGS+= -L../$(PRJ)/layout/$(INPATH)/lib
+SHL2STDLIBS += -ltklayout$(UPD)$(DLLPOSTFIX)
+.ENDIF # ENABLE_LAYOUT == TRUE
+
 # ------------------------------------------------------------------------------
 
 # Resource files
cvs diff: Diffing svx/util/defs
cvs diff: Diffing svx/workben
cvs diff: Diffing svx/workben/msview
cvs diff: Diffing svx/xml
